portfolio.css

```diff
@@ -0,0 +1,300 @@
+* {
+	box-sizing: border-box;
+	margin: 0;
+}
+
+body {
+	font-size: 62.5%;
+}
+
+.navbar {
+	display: flex;
+	justify-content: space-between;
+	height: 80px;
+}
+
+a {
+	text-decoration: none;
+	padding: 20px;
+	color: white;
+	font-size: 1.4rem;
+	font-family: "Courier New";
+}
+
+.anchors {
+	margin-top: auto;
+	margin-bottom: auto;
+}
+
+.nav-anchor2:hover {
+	border: 2px solid white;
+}
+
+.nav-anchor {
```

Comment:
might be better to target the parent container ? not sure but worth checking. in which case you can remove this class ?